README.md

```diff
@@ -144,7 +145,7 @@ take care of updates yourself.
 ### Command line arguments
 Argument | Choices / **default** | Description
 ----------|----------|----------
--b --base | **release**, nightly, beta, aurora, esr | Baseline test candidate to test against. Only used by comparative test modes.
+-b --base | **release**, nightly, beta, aurora, esr, *build tree*, *package file* | Baseline test candidate to test against. Only used by comparative test modes.
```

Comment:
Unfortunately argparser doesn't natively support such combinations of static choices and free-form elements as required for this use case, so there is already an intereactive process around advertising and validating those choices.

However, I amended the static help string for those arguments to reflect the change.